types/host.go

```diff
@@ -26,6 +26,8 @@ type Host interface {
 	CPUTimer
 	Info() HostInfo
 	Memory() (*HostMemoryInfo, error)
+
+	// FQDN returns the fully-qualified domain name of the host, lower-cased.
```

Comment:
I think the most common [form](https://www.merriam-webster.com/dictionary/lowercase) is without the dash.

```suggestion
	// FQDN returns the fully-qualified domain name of the host, lowercased.
```